docs/KIND.md

```diff
@@ -0,0 +1,34 @@
+# Run Presidio locally with KIND
```

Comment:
Where is this file referenced?

Reply:
in the main readme. But I'm now thinking on combing it with the deploy.md to have one unified md with prod/dev seperation within the file. LMK if this work

Reply:
Sounds good.